functions.sh

```diff
@@ -137,12 +137,16 @@ function get_config() {
 #
 # The result is a list of valid versions.
 function get_versions() {
+  shift
```

Comment:
I'm really bad as (ba)sh scripting, so there's probably a better solution to this. It works, tho - both with and without the version arguments